libraries/engage/locations/components/FulfillmentSelector/FulfillmentSelectorDirectShip.jsx

```diff
@@ -0,0 +1,26 @@
+// @flow
+import React from 'react';
+import { i18n } from '../../../core';
+import { Availability } from '../../../product';
+import { DIRECT_SHIP_LABEL, DIRECT_SHIP } from '../../constants';
+import { itemRow, itemColumn } from './FulfillmentSelectorItem.style';
+import { useFulfillmentSelectorState } from './FulfillmentSelector.hooks';
+
+/**
+ * Renders the direct ship item label.
+ * @returns {JSX}
+ */
+export function FulfillmentSelectorDirectShip() {
+  const { productId } = useFulfillmentSelectorState();
+
+  return (
+    <div className={itemRow}>
```

Comment:
The `Grid` from `common/components` can be used here

Reply:
Done